### PR TITLE
improvement/IISCRUM-1249

### PR DESCRIPTION
**Pull request by timyllym**
Update Iiris changes to release-3.11.0 branch

### CODE REVIEW

File: src/datasource-zabbix/module.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ ZabbixQueryController.templateUrl = 'datasource-zabbix/partials/query.editor.htm
 ZabbixDSConfigController.templateUrl = 'datasource-zabbix/partials/config.html';
 
 loadPluginCss({
-  dark: 'plugins/alexanderzobnin-zabbix-app/css/grafana-zabbix.dark.css',
-  light: 'plugins/alexanderzobnin-zabbix-app/css/grafana-zabbix.light.css'
+  dark: 'plugins/iiris-zabbix-triggers-panel/css/grafana-zabbix.dark.css',
+  light: 'plugins/iiris-zabbix-triggers-panel/css/grafana-zabbix.light.css'
```
Comment (sapajune) [inline src/datasource-zabbix/module.ts:23]: Path seems a bit weird.

**Review by sapajune — APPROVED** [review]
Looks ok.

File: src/datasource-zabbix/module.ts (hunk 1 above)
Comment (timyllym) [reply src/datasource-zabbix/module.ts:23]: Path should be ok. But something else is wrong anyway, I just tried and the datasource doesn't work.

Comment (sapajune) [reply src/datasource-zabbix/module.ts:23]: I was just referring to "triggers-panel" - seems to me this more global css than that. 😄

**Pull request merged**